rep-0151.rst

```diff
@@ -0,0 +1,293 @@
+REP: 151
+Title: Python 3 Support
+Author: Dirk Thomas, Shane Loretz
+Status: Draft
```

Comment:
Propably "Final"?

Reply:
'Final' in e6512459764898eca8ea2c9e96f30bed1861765b